docs/dagster-university/pages/dagster-dbt/lesson-6/3-creating-a-partitioned-dbt-asset.md

```diff
@@ -186,6 +196,22 @@ Here, we’ve changed the logic to say that we only want to select rows between
 
 ---
 
+## Updating the `trip_update_job` definition
+
+Before we can run the pipeline, we need to do a small bit of housekeeping. The job is partitioned monthly, and our `daily_metrics` asset is given a daily parition. It does not make sense for these to run together. So we must exclude the `daily_metrics` asset from our `trip_update_job`.  We can update this in `dagster_university/jobs/__init__.py` it to the following:
+
+```python
+daily_metrics = AssetSelection.assets("daily_metrics")
```

Comment:
`daily_metrics` is actually included in this selection:
```
dbt_trips_selection = build_dbt_asset_selection([dbt_analytics], "stg_trips").downstream()
```
As it selects `stg_trips` and the downstream assets being `daily_metrics`. So the `daily_metrics` selection is unnecessary.

<img width="1401" alt="image" src="https://github.com/user-attachments/assets/a786e914-65cb-477f-bd52-cd680fa73cb1">